homeassistant/components/modbus/__init__.py

```diff
@@ -121,6 +127,29 @@ def number(value: Any) -> int | float:
         raise vol.Invalid(f"invalid number {value}") from err
 
 
+def control_scan_interval(config: dict) -> dict:
+    """Control scan_interval."""
+    for hub in config:
+        minimum_scan_interval = DEFAULT_SCAN_INTERVAL
+        for component, conf_key in CONF_PLATFORMS:
+            if conf_key in hub:
```

Comment:
Please invert this check and `continue` if true. Then we can outdent below.

Reply:
A matter of taste, I am no friend of implicit goto, but DONE.